app/locales/ro.json

```diff
@@ -0,0 +1,112 @@
+{
+  "label": {
+    "about_title": "Despre",
+    "authorities_add_button_label": "Adaugă Sursa de Încredere",
+    "authorities_add_url": "Adaugă autoritate prin URL",
+    "authorities_desc": "Alegeti autorități de sănătate de încredere din zona dumneavoastra pentru a obține datele expunerilor. Puteți selecta un nume din registrul global, sau puteți introduce o adresa web pusa la dispoziție de institutiile care au implementat Safe Paths.",
+    "authorities_input_placeholder": "Scrie URL-ul aici",
+    "authorities_no_sources": "Nu exista încă o sursa de date",
+    "authorities_removal_alert_cancel": "Anulează",
+    "authorities_removal_alert_desc": "Sunteți sigur ca vreți sa ștergeți aceasta sursa?",
+    "authorities_removal_alert_proceed": "Șterge",
+    "authorities_removal_alert_title": "Șterge Sursa",
+    "authorities_title": "Surse de Încredere",
+    "choose_provider_subtitle": "Pentru informatii despre expuneri, trebuie sa va abonati la o autoritate de sănătate.",
+    "choose_provider_title": "Alege autoritate sănătate",
+    "commitment": "Angajament",
+    "commitment_para": "Safe Paths verifica în deplina siguranta intersectările cu alte persoane folosind locația ta. Datele tale nu vor parași NICIODATA telefonul fără aprobarea ta.",
+    "default_news_site_name": "Știri Safe Paths",
+    "default_news_site_url": "https://covidsafepaths.org/in-app-news",
+    "event_history_subtitle": "Familiarizati-va cu expunerile dumneavoastra, luând în calcul informațiile puse la dispoziție de autoritatile de sănătate",
+    "event_history_title": "Istoricul expunerilor",
+    "export_para_1": "Dacă sunteți diagnosticat pozitiv cu COVID-19, va rugam sa împărțiți istoricul traseelor pentru a ajuta la identificarea persoanelor ce au fost posibil infectate de dumneavoastra,",
+    "export_para_2": "Traseele sunt exportate ca o lista simpla ce contine latitudine, longitudine și timp, fără nici o alta informație de natura personala.",
+    "home_at_risk_header": "Expunere Posibilă",
+    "home_at_risk_subsubtext": "Asta nu înseamna ca sunteți sigur infectat,",
+    "home_at_risk_subtext": "Având în vedere istoricul dumneavoastra GPS, este posibil sa va fi intersectat cu cineva găsit pozitiv cu COVID-19.",
+    "home_enable_location": "Activați Datele Despre Locație",
+    "home_mayo_link_heading": "Mai multe informatii despre COVID-19",
+    "home_mayo_link_label": "de la Ministerul Sănătății",
+    "home_mayo_link_URL": "http://www.ms.ro/coronavirus-covid-19/",
```

Comment:
@b-dorian @tstirrat should this link take some customized local value?

Reply:
@penrods this is overwritten with non mayo link on the main screen. Is that what we want to offer as an option here?

Reply:
I would say _for now_, keep this as the Mayo Clinic link, until we decide on whether/how to include other links.

Reply:
Changed translation back to Mayo Clinic, removed URL key so it defaults